packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -842,6 +844,9 @@ class Scaffold extends StatefulWidget {
   /// The default value of this property, like the default value of
   /// [AppBar.primary], is true.
   final bool primary;
+  /// Optional callback that is called when a [Drawer] is opened or closed.
+  final DrawerCallback drawerCallback;
+  final DrawerCallback endDrawerCallback;
```

Comment:
This line also needs documentation, and there should be an empty line between every field declaration